include/NovelRT/NovelRT.h

```diff
@@ -74,55 +74,55 @@
   }
 
   //generic experimental files in root namespace
-  #include "NovelRT/EngineConfig.h"
```

Comment:
We should *probably* make these explicit `#include <NovelRT/Thing>` so that way they're using the same style users would inevitably use. Not particularly important for *this* PR though.

Reply:
This probably could get changed without much fuss in this PR. I'm indifferent either way we want to handle it.

Reply:
Still planning on fixing this